src/main/java/com/example/HR_AppJava/HR_App/HardSkill.java

```diff
@@ -1,5 +1,36 @@
 package com.example.HR_AppJava.HR_App;
 
-public class Hardskill {
-    
+import java.util.Scanner;
+
+public class HardSkill {
+
+    Scanner scanner = new Scanner(System.in);
+
+    private String title;
+    private String level;
+
+    public HardSkill(){
```

Comment:
You shouldn't call your modifier methods inside of your constructor. For the default constructor, you just need 
public Person(){}
You could also add a constructor that takes parameters as
public HardKill(String title, String level){
this.title = title;
this.level = level;
}